### PR TITLE
Label odd numbered tests

### PR DESCRIPTION
**Pull request by matetokodi**

### CODE REVIEW

**Comment by zherczeg** [comment]
Please always squash your commits

File: test_src/sljitTest.c [hunk 1]
```diff
@@ -306,10 +306,16 @@ static void test3(void)
 	buf[4] = 0x12345678;
 
 	sljit_emit_enter(compiler, 0, SLJIT_ARGS1(W, P), 3, 1, 0, 0, 0);
+
```
Comment (zherczeg) [inline test_src/sljitTest.c:309]: No need these extra lines, the test becomes too long.

File: test_src/sljitTest.c [hunk 2]
```diff
@@ -735,44 +781,61 @@ static void test9(void)
...
+	/*buf[4] */
+	/*buf[5] */
 	sljit_emit_op1(compiler, SLJIT_MOV, shift_reg, 0, SLJIT_IMM, 0xff);
 	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_IMM, 8);
 	sljit_emit_op2(compiler, SLJIT_LSHR, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 4, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 4, SLJIT_R0, 0);
 	sljit_emit_op2(compiler, SLJIT_SHL, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 5, shift_reg, 0, SLJIT_R0, 0);
 
+	/*buf[6] */
+	/*buf[7] */
```
Comment (zherczeg) [inline test_src/sljitTest.c:814]: You can write this in one line.

**Review by zherczeg — CHANGES_REQUESTED** [review]
Good patch

**Review by zherczeg — CHANGES_REQUESTED** [review]
Good patch. Not entirely clear when newlines are added or removed. Please fix the conflicts.

**Comment by matetokodi** [comment]
Updated to keep the original spacing, except in test69, where I feel it is redundant with the addition of the comments

File: test_src/sljitTest.c [hunk 3]
```diff
@@ -735,44 +758,54 @@ static void test9(void)
 	buf[12] = 0;
 
 	sljit_emit_enter(compiler, 0, SLJIT_ARGS1(VOID, P), 4, 2, 0, 0, 0);
+	/*buf[0] */
```
Comment (zherczeg) [inline test_src/sljitTest.c:761]: Somehow from this point there is no space between the `*` and buf. Several similar cases below.

File: test_src/sljitTest.c [hunk 4]
```diff
@@ -735,44 +758,54 @@ static void test9(void)
...
 	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_IMM, -64);
 	sljit_emit_op1(compiler, SLJIT_MOV, shift_reg, 0, SLJIT_IMM, 2);
 	sljit_emit_op2(compiler, SLJIT_ASHR, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 2, SLJIT_R0, 0, shift_reg, 0);
 
+	/*buf[3] */
 	sljit_emit_op1(compiler, SLJIT_MOV, shift_reg, 0, SLJIT_IMM, 0xff);
 	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_IMM, 4);
 	sljit_emit_op2(compiler, SLJIT_SHL, shift_reg, 0, shift_reg, 0, SLJIT_R0, 0);
 	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw) * 3, shift_reg, 0);
+	/*buf[4] and buf[5] */
```
Comment (zherczeg) [inline test_src/sljitTest.c:779]: The other patch used this format (sorry that was landed first):
`/* buf[4-5] */`
Please do this as well for consistency. I saw you also followed this format in some places.

File: test_src/sljitTest.c [hunk 5]
```diff
@@ -973,9 +1011,13 @@ static void test11(void)
 	FAILED(buf[1] != -65535, "test11 case 3 failed\n");
 	FAILED(buf[2] != word_value1, "test11 case 4 failed\n");
 
+	/*buf[0] */
 	sljit_set_const(const1_addr, -1, executable_offset);
+	/*buf[1] */
 	sljit_set_const(const2_addr, word_value2, executable_offset);
+	/*buf[2] */
 	sljit_set_const(const3_addr, (sljit_sw)0xbab0fea1, executable_offset);
+	/*Return vaue */
```
Comment (zherczeg) [inline test_src/sljitTest.c:1020]: There is no space before `Return` word.

**Review by zherczeg — CHANGES_REQUESTED** [review]
I like that even return values are tagged with comments.

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**